### ISSUE TITLE
Unable to activate plugin: class Maintenance_Mode_WP does not have a method "activate"

### ISSUE THREAD

**Issue by urosevic** [robertdevore/maintenance-mode-for-wordpress#1]
Hi Robert,

I tried 1.0.0 on fresh WordPress 6.7.1 with the Twenty Fifteen 3.9 theme, and PHP 8.3.13 and 8.2.25. Activating failed as there is no `activate` static method in the `Maintenance_Mode_WP ` class called [on line 31](https://github.com/robertdevore/maintenance-mode-for-wordpress/blob/940c48df76da109695dacd8afd79cb3a2cee519a/maintenance-mode-for-wordpress.php#L31):

```
[09-Dec-2024 10:58:45 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, class Maintenance_Mode_WP does not have a method "activate" in C:\www\latest.wp\wp-includes\class-wp-hook.php:324
Stack trace:
#0 C:\www\latest.wp\wp-includes\class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 C:\www\latest.wp\wp-includes\plugin.php(517): WP_Hook->do_action(Array)
#2 C:\www\latest.wp\wp-admin\includes\plugin.php(703): do_action('activate_mainte...', false)
#3 C:\www\latest.wp\wp-admin\plugins.php(60): activate_plugin('maintenance-mod...', 'https://latest....', false)
#4 {main}
  thrown in C:\www\latest.wp\wp-includes\class-wp-hook.php on line 324
```

**Comment by alex-borto** [comment]
Same error from my side here. Once it will be fixed, I'll shoot a video for WPMarmite's YouTube channel :)

**Comment by robertdevore** [comment]
This has been fixed in v1.0.1 which has just been pushed out 🤘

Thank you both 🙏
